packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -63,16 +67,23 @@ const selectors = {
 export class EditorToolbarComponent {
 	private page: Page;
 	private editor: Locator;
+	private header: string;
 
 	/**
 	 * Constructs an instance of the component.
 	 *
 	 * @param {Page} page The underlying page.
 	 * @param {Locator} editor Locator or FrameLocator to the editor.
+	 * @param {string} context Context of the editor (e.g. whether it's editing a post or the site).
 	 */
-	constructor( page: Page, editor: Locator ) {
+	constructor(
+		page: Page,
+		editor: Locator,
+		context: 'post-editor' | 'site-editor' = 'post-editor'
```

Comment:
**nit:blocking** - the term `context` is quite loaded in Playwright, as it refers to the browsercontext (think of it as a session containing multiple pages which are tabs). 

Would something like `editorType` work instead?

Reply:
I think these changes are no longer needed because of https://github.com/Automattic/wp-calypso/pull/72112, so I'll see if I can revert them as soon as we're able to switch the site's theme back to a block theme.

Reply:
I was able to revert these changes thanks to https://github.com/Automattic/wp-calypso/pull/72112.